recipe/meta.yaml

```diff
@@ -5,9 +5,9 @@ package:
   name: {{ name|lower }}
   version: {{ version }}
 
-source:
-  url: http://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
-  sha256: 532c5387270dd55ff1e676e929a77c5910f2fba92fa62885b1f5ed0aef6fdde8
+source: 
```

Comment:
@Kwaizer so that you don't add whitespace to the end of lines can you enable "Trim Trailing Whitespace" in your VS Code settings? If you go to User Settings it is under "Text Editor" -> "Files" and then scroll down to "Trim Trailing Whitespace".

![image](https://github.com/user-attachments/assets/27aabb2d-2055-44fb-b055-78b4f0c62d49)